partner_contact_birthplace/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Rubino - Agile Business Group
+# Copyright 2019 Nicolò Crisafulli - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner contact birthplace',
+    'summary': 'This module allows to define a birthplace for partners.',
+    'version': '12.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'website': 'https://github.com/OCA/partner-contact/tree/12.0/'
```

Comment:
Here Only  `https://github.com/OCA/partner-contact`

Reply:
Done

Reply:
@nikul-serpentcs There is no issue with it. We can provide either full addons path or repository name. Ref: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

Reply:
@Crisa221 :  As per your choice, would you please make small correction.

Reply:
> @nikul-serpentcs Your comment is invalid to me. We can provide either full addons path or repository name. Ref: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

Yes @bodedra It's Not a big deal, Either Only Repo. Path Or Full. But Mostly Developer Chose only Repo. path.